internal/daemon/controller/rewrapping_test.go

```diff
@@ -0,0 +1,58 @@
+package controller
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/boundary/internal/db"
+	"github.com/hashicorp/boundary/internal/kms"
+	"github.com/stretchr/testify/assert"
+)
+
+func SliceContains(slice []string, item string) bool {
+	for _, i := range slice {
+		if item == i {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
Nit: you can use https://pkg.go.dev/golang.org/x/exp/slices#Index and check that `i > 0`.

Reply:
:doh: there's also just https://pkg.go.dev/golang.org/x/exp/slices#Contains

Reply:
yknow. I actually looked for something like this, and didn't find anything. sighhhhhhhhh

Reply:
moot point with cmp.diff